### PR TITLE
multiplier function without using *

### PR DESCRIPTION
**Pull request by aidandsheehy**
See title

### CODE REVIEW

File: Aidan/multiply.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                prod = prod + num2
+        else:
+            for ii in range(num1, 0):
+                prod = prod - num2
+    return prod
+   
+
+
+result = multiplier(5,0)
+print(result)
```
Comment (WillGolf4Food) [inline Aidan/multiply.py]: Let's add a few test cases here so we can watch all the tests run.  See my additions ...

File: Aidan/multiply.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#! C:\Users\HP\local repository\AWS\Aidan
```
Comment (WillGolf4Food) [inline Aidan/multiply.py]: Let's change this line to make the file executable.

Comment (aidandsheehy) [reply Aidan/multiply.py]: Copy that

File: Aidan/multiply.py (hunk 1 above)
Comment (aidandsheehy) [reply Aidan/multiply.py]: Understood!

**Pull request merged**